withdraw-by-check/withdraw-by-check-json-contract.md

```diff
@@ -0,0 +1,99 @@
+# Withdraw by check JSON contract
+To get things started, the client will send an initial request structured like so:
+```json
+{
+  "rgState": "STATESTART",
+  "powerOnFileName": "BANNO.WITHDRAW.CHECK.V1.POW",
+  "userChrList":[
+    {"id": 1, "value": "member-account-number"},
+    {"id": 2, "value": "4charLoanId"},
+    {"id": 3, "value": ""},
+    {"id": 4, "value": ""},
+    {"id": 5, "value": ""}
+  ],
+  "userNumList": [],
+  "rgSession": 1
+}
+```
+If successful, the poweron should respond with:
+```json
+{
+  "results": {
+    "shareEligible": true,
+    "accountNumber": "0123456789",
+    "accountDescription": "ADVANCED CHECKING",
+    "available": "123456.00",
+    "owner": "Julie Jones",
+    "address": [
+      "Julie Jones",
+      "6525 Chancellor Drive",
+      "Cedar Falls",
+      "IA",
+      "50613",
+      "ADDRESS 6"
+    ],
+    "disclaimerText": ["disclaimer ", "lines ", "with ", "spaces."]
+  }
+}
+```
+### Errors
+Missing or invalid Letterfile:
+```json
+{
+  "errorCode": 500,
+  "loggingErrorMessage": "Error Opening Letterfile BANNO.WITHDRAW.CHECK.V1.CFG: No such file or directory"
```

Comment:
For the logging error, should we be more specific  - by including the actual host error?

Reply:
I'll leave the content up to you.